pkg/pull/pull.go

```diff
@@ -391,21 +391,24 @@ func Pull(upstreamURI string, pullOptions PullOptions) (string, error) {
 	}
 
 	helmMidstreams := []midstream.Midstream{}
-	for _, base := range helmBases {
+	for _, helmBase := range helmBases {
 		writeMidstreamOptions := commonWriteMidstreamOptions
-		writeMidstreamOptions.MidstreamDir = filepath.Join(base.GetOverlaysDir(writeBaseOptions), "midstream", base.Path)
-		writeMidstreamOptions.BaseDir = filepath.Join(u.GetBaseDir(writeUpstreamOptions), base.Path)
-
-		helmBase := base
```

Comment:
are you sure you can remove this?

Reply:
I renamed `base` variable to not be the same as `base` package.  It's now created as `helmBase`.